common/tl/query-header.cpp

```diff
@@ -98,7 +98,7 @@ static int tl_fetch_query_flags(tl_query_header_t *header) {
   return 0;
 }
 
-bool tl_fetch_query_header(tl_query_header_t *header) {
```

Comment:
Seams to be a strange hack. Is this stat really useful? Maybe just drop it?

Reply:
Anyway, I think it was a bug, it was not incremented before

Reply:
removed this stat completely